controllers/goharbor/harborcluster/database/api/postgresql_type.go

```diff
@@ -0,0 +1,211 @@
+package api
+
+// Postgres CRD definition, please use CamelCase for field names.
+
+import (
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Postgresql defines PostgreSQL Custom Resource Definition Object.
+type Postgresql struct {
```

Comment:
Why need to rewrite the Postgresql CRD definition? Package import is not worked?

Reply:
yes, The introduction of official CRD will cause version conflicts.